tests/shared/infra/repositories/test_action_repository_mock.py

```diff
@@ -35,6 +35,7 @@ def test_get_all_actions_by_ra(self):
         assert len(actions) == len(expected)
         assert actions == expected
         assert all([type(action) == AssociatedAction for action in actions])
+        assert all([action.member_ra for action in actions])
```

Comment:
all([action.member_ra == ra_escolhido for action in actions])

Reply:
ok